ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Util/MonadSTM/RAWLock.hs

```diff
@@ -1,10 +1,11 @@
+{-# LANGUAGE DataKinds                  #-}
 {-# LANGUAGE DeriveAnyClass             #-}
 {-# LANGUAGE DeriveGeneric              #-}
-{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE DerivingVia                #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE LambdaCase                 #-}
 {-# LANGUAGE ScopedTypeVariables        #-}
--- | A Read-Append-Write (RAW) lock
+-- | A writer-biased Readers-Appender-Writer (RAW) lock
```

Comment:
Any particular reason for this change? Other places still talk about "Read-Append-Write".

Reply:
Well, it is `write-biased` 😄

Reply:
Oh yeah, that part makes sense, was only commenting about the consistency of "Readers-Appender-Writer" vs "Read-Append-Write".

Reply:
Rewritten. Is now ok?